api-report/driver-definitions.api.md

```diff
@@ -200,6 +200,7 @@ export interface IDriverBasicError extends IDriverErrorBase {
 // @public
```

Comment:
(leaving comment here for easier commenting, but it applies to whole PR).
How do we test these changes? There is a chance that we will make it worse, especially for offline (by mistake), or online (where the decision to go with slower retries may backfire somewhere as we can't anticipate situation)?

I think at minimal we should discuss it with Loop falls to get at least some feedback.
Adding @AndreiZe to provide feedback.
FYI - I've updated PR description to add more context.

Reply:
The specific issue that we are trying to address is unnecessary spamming of service which is under load. The service is supposed to return correct retry-after header, but we have experienced the event where service was not doing it correctly.
I would encourage being super defensive here in order to avoid hitting app level throttling which did happen in case of aforementioned event. I would suggest backing off to upper limit of 2 minutes and then making it constant.

Reply:
I'd suggest that we do it slowly. I.e. right now we have 8 seconds max between retries. I do not know what would break if we go with 2 minutes. So, I'd suggest going to 30 seconds, then maybe in a month - to a minute, and then to two.

Reply:
hmm ok. Lets go with 30s first. We can increase later.